packages/flutter_tools/test/commands.shard/hermetic/build_ios_test.dart

```diff
@@ -206,16 +220,18 @@ void main() {
   }, overrides: <Type, Generator>{
     Platform: () => macosPlatform,
     FileSystem: () => fileSystem,
-    ProcessManager: () => FakeProcessManager.any(),
+    ProcessManager: () => processManager,
```

Comment:
Thanks for making this more strict :)

Reply:
i'd like to delete FakeProcessManager.any(), except for the work that would entail...